horizon/templates/app-credentials.secret.yaml

```diff
@@ -15,9 +15,9 @@ data:
   POSTGRES_PASSWORD: {{ .Values.dependencies.postgresql.password | b64enc }}
   {{- end }}
   OPENNMS_DBUSER: {{ .Values.core.configuration.database.username | b64enc }}
-  OPENNMS_DBPASS: {{ .Values.core.configuration.database.password | b64enc }}
+  OPENNMS_DBPASS: {{ required "Database password (core.configuration.database.password) must be provided" .Values.core.configuration.database.password | b64enc }}
```

Comment:
It would be good to let this be empty for local development databases (see https://github.com/OpenNMS/helm-charts/tree/main/examples/minikube-host-postgresql.yaml)